packages/account/src/Sections/Security/Passkeys/__tests__/passkeys.spec.tsx

```diff
@@ -159,173 +164,137 @@ describe('Passkeys', () => {
 
-        (useRegisterPasskey as jest.Mock).mockReturnValue({
-            createPasskey: mockCreatePasskey,
-            is_passkey_registration_started: true,
-            startPasskeyRegistration: mockStartPasskeyRegistration,
+        (useRemovePasskey as jest.Mock).mockReturnValue({
+            removePasskey: mockUseRemovePasskey,
         });
 
         renderComponent();
 
-        userEvent.click(screen.getByRole('button', { name: create_passkey }));
-        expect(screen.getByText('Just a reminder')).toBeInTheDocument();
-        expect(screen.getByText('Enable screen lock on your device.')).toBeInTheDocument();
-        expect(screen.getByText('Enable bluetooth.')).toBeInTheDocument();
-        expect(screen.getByText('Sign in to your Google or iCloud account.')).toBeInTheDocument();
+        userEvent.click(screen.getAllByTestId('dt_dropdown_display')[0]);
 
-        userEvent.click(screen.getByRole('button', { name: continue_button }));
-        expect(mockCreatePasskey).toBeCalledTimes(1);
-        expect(Analytics.trackEvent).toHaveBeenCalledWith(
-            tracking_event,
-            getAnalyticsParams('create_passkey_reminder_passed')
-        );
+        await waitFor(() => {
+            expect(screen.getByText('Rename')).toBeInTheDocument();
+            expect(screen.getByText('Remove')).toBeInTheDocument();
+        });
+
+        userEvent.click(screen.getByText('Remove'));
+        expect(mockUseRemovePasskey).toHaveBeenCalledTimes(1);
+        expect(Analytics.trackEvent).toHaveBeenCalledWith(tracking_event, getAnalyticsParams('passkey_remove_started'));
     });
 
-    it('renders passkeys registration error modal and triggers closing', async () => {
-        (useRegisterPasskey as jest.Mock).mockReturnValue({
-            passkey_registration_error: { message: 'error' },
-            clearPasskeyRegistrationError: mockClearPasskeyRegistrationError,
-            startPasskeyRegistration: mockStartPasskeyRegistration,
-            createPasskey: mockCreatePasskey,
+    it('should trigger renaming of passkey ', async () => {
+        (useGetPasskeysList as jest.Mock).mockReturnValue({
+            passkeys_list: mock_passkeys_list,
+        });
+
+        const mockUseRenamePasskey = jest.fn();
+        (useRenamePasskey as jest.Mock).mockReturnValue({
+            renamePasskey: mockUseRenamePasskey,
         });
 
         renderComponent();
 
-        userEvent.click(screen.getByRole('button', { name: create_passkey }));
-        userEvent.click(screen.getByRole('button', { name: continue_button }));
+        userEvent.click(screen.getAllByTestId('dt_dropdown_display')[0]);
 
         await waitFor(() => {
-            expect(screen.getByText(error_message)).toBeInTheDocument();
-            expect(screen.getByText(error_title)).toBeInTheDocument();
+            expect(screen.getByText('Rename')).toBeInTheDocument();
+            expect(screen.getByText('Remove')).toBeInTheDocument();
```

Comment:
how about adding a test case for"Remove" flow

Reply:
the previous test is to check, that removing flow is triggered.
after clicking 'remove' button, authenticator api is triggered and call to BE happens. 
so we can't test it here, it will be just mocks.